scripts/check_integration.sh

```diff
@@ -8,6 +8,7 @@ set -e
 
 declare -a tests=(
    "healthcheck"
+   "issuer_adapter"
```

Comment:
ah - so this is why I didn't see the rp bdd tests run in the azure pipeline. Can you add those as well?

Reply:
Added [here](https://github.com/trustbloc/edge-adapter/pull/80/files#diff-efd428097dcdd2a893685ab7440126aeR12).